### PR TITLE
latte-assembler: fixes/improvements.

### PR DESCRIPTION
**Pull request by aliaspider**

### CODE REVIEW

File: tools/latte-assembler/src/gfd.cpp [hunk 1]
```diff
@@ -325,7 +493,7 @@ gfdAddPixelShader(gfd::GFDFile &file,
 
    out.regs.sq_pgm_resources_ps = out.regs.sq_pgm_resources_ps
       .NUM_GPRS(numGpr)
-      .STACK_SIZE(1);
+      .STACK_SIZE(out.loopVars.size() * 2);
```
Comment (aliaspider) [inline tools/latte-assembler/src/gfd.cpp]: I'm still not entirely sure about this one. there is definitely some correlation between loopVarsCount and stacksize though.

**Comment by brett19** [comment]
I see Geometry Shader related stuff here, there may be useful information here as well:
https://github.com/decaf-emu/decaf-emu/tree/geom

**Comment by aliaspider** [comment]
the assembler doesn't support geometry shaders yet, those changes you saw were only for libgfd/gfd-tool.

**Comment by exjam** [comment]
Merged, thanks.

**Pull request closed without merging**